core/modules/ckeditor/ckeditor.module

```diff
@@ -4,7 +4,7 @@
  * Provides integration with the CKEditor WYSIWYG editor.
  */
 
-define('CKEDITOR_VERSION', '4.16.2');
+define('CKEDITOR_VERSION', '4.17.1');
```

Comment:
Touching this is not necessary. The current version of our CKEditor is 4.18.0, BTW.